modules/nf-core/dragen/main.nf

```diff
@@ -0,0 +1,78 @@
+process DRAGEN {
+    tag "$meta.id"
+    label 'process_long'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE':
+        'registry.hub.docker.com/etycksen/dragen4:4.2.4' }" // somehow the PATH is not containing dragen, when I run it locally it is there
+
+    input:
+    tuple val(meta), path(fastq)
```

Comment:
There is several other inputs but I assumed we start with a single use case `--enable-variant-caller true`

Reply:
the `anable-variant-caller true` requires `--RGID` and `--RGSM`. How should we set them - does the user need to provide something, should they default to `prefix`?